### PR TITLE
Add RotMatrix2, RotMatrix3 type aliases.

### PR DESCRIPTION
**Pull request by tkoolen**
I've found these come in handy.

### CODE REVIEW

**Comment by tkoolen** [comment]
Unrelated failure on OSX nightly, I believe.

File: test/rotation_tests.jl [hunk 1]
```diff
@@ -261,4 +261,9 @@ all_types = (RotMatrix{3}, Quat, SPQuat, AngleAxis, RodriguesVec,
             end
         end
     end
+
+    @testset "Testing type aliases" begin
+        @test typeof(eye(RotMatrix{2, Float64})) == RotMatrix2{Float64}
+        @test typeof(eye(RotMatrix{3, Float64})) == RotMatrix3{Float64}
```
Comment (c42f) [inline test/rotation_tests.jl]: This can be a nice place to use `isa`:

```
@test eye(RotMatrix{2, Float64}) isa RotMatrix2{Float64}
```

Comment (tkoolen) [inline test/rotation_tests.jl]: Done.

**Comment by c42f** [comment]
The latest OSX failure is unrelated for sure. Restarted it for good measure, and now it seems to have gone through.

**Pull request merged**